resources/g2/sprites.json

```diff
@@ -1223,5 +1223,5 @@
         "y_offset": 4,
         "palette": "keep",
         "forceBmp": true
-    }
+    },
```

Comment:
The travis builds fail because of this new comma. I think you meant to add the new image here.